packages/yoga/src/Heading/Heading.theme.js

```diff
@@ -3,11 +3,13 @@ const Heading = ({ spacing, colors }) => ({
   padding: {
     vertical: spacing.small,
     horizontal: spacing.huge,
+    horizontalMobile: spacing.large,
```

Comment:
The prototype have `20px` of horizontal padding.

```suggestion
    horizontalMobile: spacing.medium,
```